packages/components/addon/components/hds/form/textarea/base.hbs

```diff
@@ -1,2 +1,2 @@
 {{! Notice: this is not the native HTML <textarea> but the Ember component <Textarea> }}
-<Textarea class={{this.classNames}} {{style width=@width}} ...attributes @value={{@value}} />
\ No newline at end of file
+<Textarea class={{this.classNames}} {{style width=@width}} rows="{{this.rows}}" ...attributes @value={{@value}} />
```

Comment:
I think we can keep it much simpler! Just do this:
```suggestion
<Textarea class={{this.classNames}} {{style width=@width}} rows="4" ...attributes @value={{@value}} />
```
and you can remove all the code below. If a user pass a `rows` attribute, the `...attributes` spread will override the default.